### PR TITLE
Detect launch and landing

### PR DESCRIPTION
**Pull request by JEK58**

### CODE REVIEW

File: server/igc/IgcAnalyzer.js [hunk 1]
```diff
@@ -52,8 +53,16 @@ const IgcAnalyzer = {
 
     const igcAsPlainText = readIgcFile(flightDataObject);
 
-    //IGCParser needs lenient: true because some trackers (e.g. XCTrack) work with addional records in IGC-File which don't apply with IGCParser.
+    // IGCParser needs lenient: true because some trackers (e.g. XCTrack) work with addional records in IGC-File which don't apply with IGCParser.
     const igcAsJson = IGCParser.parse(igcAsPlainText, { lenient: true });
+
+    // Remove non flight fixes
+    const launchAndLanding = findLaunchAndLanding(igcAsJson);
```
Comment (KaiWissel) [inline server/igc/IgcAnalyzer.js:60]: why is this an array?

File: server/igc/IgcHelper.ts [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+        )
+          for (let j = start; j <= i; j++) fixes[i].stateGround = true;
+      } else {
+        start = undefined;
+      }
+  }
+}
+
+export function detectLaunchLanding(fixes: ExtendedBRecord[]) {
+  let ll = [];
```
Comment (KaiWissel) [inline server/igc/IgcHelper.ts:159]: ll is the short for???

File: server/igc/IgcAnalyzer.js (hunk 1 above)
Comment (JEK58) [reply server/igc/IgcAnalyzer.js:60]: It can detect multiple flights in the same igc file.

File: server/igc/IgcHelper.ts (hunk 2 above)
Comment (JEK58) [reply server/igc/IgcHelper.ts:159]: launch & landing (I didn't name it like that. It comes from the original implementation.)

**Pull request merged**